src/dmd/traits.d

```diff
@@ -736,7 +736,7 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
             return dimError(1);
 
         Scope* sc2 = sc.push();
-        sc2.flags = sc.flags | SCOPE.noaccesscheck;
+        sc2.flags = sc.flags | SCOPE.noaccesscheck | SCOPE.ignoresymbolvisibility;
```

Comment:
Wasn't that a bug ? Also do we still need `noaccesscheck`?

Reply:
Wasn't what a bug?  I'll look into removing the `noaccesscheck`.

Reply:
I couldn't remove `noaccesscheck`.  If I do `runnable/traits.d` emits failures.